src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,115 @@
+.clock {
+  position: relative;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  @include clock-circle($watch-size);
+
+  background-color: beige;
+  border: 10px solid $clock-color;
+
+  &__top {
+    position: absolute;
+    right: 50%;
+
+    width: $indicator-size;
+    height: $indicator-size * 2;
+    background-color: $clock-color;
+  }
+
+  &__right {
+    position: absolute;
+    top: 50%;
+    right: 0;
+
+    width: $indicator-size * 2;
+    height: $indicator-size;
+    background-color: $clock-color;
+  }
+
+  &__bottom {
+    position: absolute;
+    right: 50%;
+    bottom: 0;
+
+    width: $indicator-size;
+    height: $indicator-size * 2;
+    background-color: $clock-color;
+  }
+
+  &__left {
+    position: absolute;
+    top: 50%;
+    left: 0;
+
+    width: $indicator-size * 2;
+    height: $indicator-size;
+    background-color: $clock-color;
+  }
```

Comment:
Don't repeat yourself, these three selectors have the same rules, let's group them. 
left, right, and bottom you could set separately

Reply:
done